numpy/add_newdocs.py

```diff
@@ -5822,7 +5828,9 @@ def luf(lamdaexpr, *args, **kwargs):
         the data type of the input array if no output array is provided.
     out : ndarray, optional
         A location into which the result is stored. If not provided a
-        freshly-allocated array is returned.
+        freshly-allocated array is returned. A tuple of arrays
+        (possible only as a keyword argument) must have length equal to the
+        number of outputs; use `None` for outputs to be allocated by the ufunc.
```

Comment:
This is a bit wordy - can you combine "use `None` for outputs to be allocated by the ufunc" with "If not provided a freshly-allocated array is returned. "? Not provided and none are the same thing with the default documented as it is.